### PR TITLE
C#: Add random functions to GD class

### PR DESCRIPTION
**Pull request by neikeq**
These were missing for a quite some time.

### CODE REVIEW

File: modules/mono/glue/Managed/Files/GD.cs [hunk 1]
```diff
@@ -110,6 +110,31 @@ public static void PrintT(params object[] what)
...
+        {
+            return godot_icall_GD_randi();
+        }
+
+        public static void Randomize()
+        {
+            godot_icall_GD_randomize();
+        }
+
+        public static double rand_range(double from, double to)
```
Comment (akien-mga) [inline modules/mono/glue/Managed/Files/GD.cs]: Shouldn't it be `RandRange`?

File: modules/mono/glue/Managed/Files/GD.cs [hunk 2]
```diff
@@ -110,6 +110,31 @@ public static void PrintT(params object[] what)
...
+        {
+            return godot_icall_GD_rand_range(from, to);
+        }
+
+        public static uint RandSeed(ulong seed, out ulong newSeed)
+        {
+            return godot_icall_GD_rand_seed(seed, out newSeed);
```
Comment (akien-mga) [inline modules/mono/glue/Managed/Files/GD.cs]: Shouldn't it be `godot_icall_GD_rand_from_seed` and `RandFromSeed`, as in core? The current name suggests that it returns a random seed, while it returns a random number from the given seed, and also gives the updated seed.

Comment (neikeq) [reply modules/mono/glue/Managed/Files/GD.cs]: I agree `RandFromSeed` would be better but the current name matches the GDScript one.

File: modules/mono/glue/Managed/Files/GD.cs (hunk 1 above)
Comment (neikeq) [reply modules/mono/glue/Managed/Files/GD.cs]: Yes... I don't know how I missed that.

Comment (neikeq) [reply modules/mono/glue/Managed/Files/GD.cs]: Fixed by #26065

**Pull request merged**